pmd-core/src/main/java/net/sourceforge/pmd/renderers/XMLRenderer.java

```diff
@@ -46,6 +46,7 @@
 
     private XMLStreamWriter xmlWriter;
     private OutputStream stream;
+    private byte[] lineSeparator = new byte[0];
```

Comment:
this initialization seems moot, right?

```suggestion
    private byte[] lineSeparator;
```

Reply:
yes, that's not necessary.